wgsl/index.bs

```diff
@@ -73,8 +73,8 @@ TODO: What indicates the end of a line?  (E.g. A line ends at the next linefeed
   <thead>
     <tr><td>Token<td>Definition
   </thead>
-  <tr><td>`FLOAT_LITERAL`<td>`(-?[0-9]*.[0-9]+ | -?[0-9]+.[0-9]*)((e|E)(+|-)?[0-9]+)?`
-  <tr><td>`INT_LITERAL`<td>`-?0x[0-9a-fA-F]+ | 0 | -?[1-9][0-9]*`
+  <tr><td>`FLOAT_LITERAL`<td>`(MINUS? [0-9]* PERIOD [0-9]+ | MINUS? [0-9]+ PERIOD [0-9]*)((e|E)(PLUS|MINUS)?[0-9]+)?`
```

Comment:
We should probably do this anyway, can you pull this out to its own CL?

Reply:
There really should not be a leading MINUS in a FLOAT_LITERAL, negation is part of parsing, not part of lexing.  It makes the existence of white space matter in the HLL, when it should not.  (E.g., need for space becomes asymmetric around +, -, *, /, whereas this is not true in other C-like languages).

Reply:
This is a separate issue, not related to this CL. Changing this has not been discussed by the community group at this point.